app/assets/stylesheets/show (Alex’s MacBook Air's conflicted copy 2016-01-15).scss

```diff
@@ -0,0 +1,99 @@
+.show-container {
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border-radius, margin, padding, position